lib/web_ui/lib/src/engine/view_embedder/dom_manager.dart

```diff
@@ -63,4 +63,24 @@ class DomManager {
   /// Otherwise, the phone will disable focusing by touch, only by tabbing
   /// around the UI.
   DomElement get semanticsHost => _embedder.semanticsHostElementDEPRECATED;
+
+  DomElement get _sceneHost => _embedder.sceneHostElementDEPRECATED;
+
+  DomElement? _lastSceneElement;
+
+  /// Inserts the [sceneElement] into the DOM.
+  ///
+  /// The [sceneElement] is inserted  as a child of the <flt-scene-host> element
+  /// inside the [renderingHost].
+  ///
+  /// If the [sceneElement] has already been inserted, this method does nothing
+  /// to avoid unnecessary DOM mutations. This is both faster and more correct,
+  /// because moving DOM nodes loses internal state, such as text selection.
+  void insertScene(DomElement sceneElement) {
```

Comment:
I'm not sure `insert` is the right name (to me it means "add anywhere but keep the old stuff?).

However the only thing I can come up with is `replace` and I don't like it either. (This is private API and can be renamed later very easily so, let's let it gggo! ❄️)

Reply:
Maybe `setScene`?

Reply:
Yes!